### ISSUE TITLE
Is there any way to make tts bot ignore the message?

### ISSUE THREAD

**Issue by Velover** [Discord-TTS/Bot#109]
something like adding special characters at the start of the message for tts to ignore it

sometimes it can get annoying that when you share some data and the tts bot starts speaking it aloud

what i'm talking of is like

- This message will be read
- ///This message will be ignored

**Comment by GnomedDev** [comment]
Yes, if you start a message with the command prefix (by default, `-`) it will ignore it.

**Comment by Velover** [comment]
thank you
